webapp/src/app/shared/phaser/map-pan.ts

```diff
@@ -81,11 +81,11 @@ export class MapPan extends Phaser.GameObjects.GameObject {
 	
 	preUpdate() {
 
-		Globals.panning = this.panKey.isDown || this.scene.input.activePointer.middleButtonDown();
-
 		const pointer = this.scene.input.activePointer;
+		
+		Globals.panning = this.panKey.isDown || pointer.middleButtonDown();
 
-		if (Globals.panning && this.isScrolling && this.active) {
+		if (Globals.panning && this.isScrolling && this.active && pointer.isDown) {
```

Comment:
@Vegita2 Found a case where onMouseUp wasn't being called when the pointer was off the screen.  When this occurs, `isScrolling` doesn't ever get updated.  Pressing ctrl in this case would move the entire camera, even though scrolling is not active.  Added `pointer.isDown` to prevent this from happening.